File: 프로그래머스/0/120860. 직사각형 넓이 구하기/직사각형 넓이 구하기.cs
using System;
using System.Collections.Generic;

public class Solution 
{
    public int solution(int[,] dots) 
    {
        List<int> xPos = new List<int>();
        List<int> yPos = new List<int>();
        for(int i=0; i<dots.GetLength(0); i++)
        {
            xPos.Add(dots[i,0]);
        }
        for(int i=0; i<dots.GetLength(0); i++)
        {
            yPos.Add(dots[i,1]);
        }
        xPos.Sort();
        yPos.Sort();
        
        return Math.Abs(xPos[1] - xPos[2]) * Math.Abs(yPos[1] - yPos[2]);
    }
}

File: 프로그래머스/0/120860. 직사각형 넓이 구하기/직사각형 넓이 구하기.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

int solution(vector<vector<int>> dots) 
{
    vector<int> x;
    vector<int> y;
    
    for(auto &v : dots)
    {
        x.push_back(v[0]);
        y.push_back(v[1]);
    }
    sort(x.begin(), x.end());
    sort(y.begin(), y.end());
    int area = (x[2] - x[1]) * (y[2] - y[1]);
    return area;
}